MojioSDK/Client/Core/RestClient.swift

```diff
@@ -120,16 +113,19 @@ open class RestClient {
     
     public let sessionManager: SessionManager
     internal var keychainManager: KeychainManager
+    private let arrayEncoding: URLEncoding.ArrayEncoding
```

Comment:
Can we make `arrayEncoding` public, add a default value and if needed add methods to change the value as we do for `requestMethod`?

Reply:
@andriimakukha It makes sense. But frankly saying we didn't use this option much. Actually it's the first time we're using that... So, I think it's a minor thing for this moment